cognite/client/_api/data_modeling/instances.py

```diff
@@ -214,7 +214,7 @@ def __call__(
         instance_type: Literal["node", "edge"] = "node",
         limit: int | None = None,
         include_typing: bool = False,
-        sources: list[ViewId] | ViewId | None = None,
+        sources: list[ViewId] | ViewId | list[View] | list[ViewId] | None = None,
```

Comment:
```suggestion
        sources: list[ViewId] | ViewId | list[View] | None = None,
```

Reply:
oups was supposed to be `list[View] | View`